frontend/public/components/network-policy.jsx

```diff
@@ -178,26 +184,29 @@ const IngressRow = ({ ingress, namespace, podSelector }) => {
 };
 
 const Details_ = ({ obj: np, flags }) => {
+  const { t } = useTranslation();
   return (
     <>
       <div className="co-m-pane__body">
-        <SectionHeading text="Namespace Details" />
+        <SectionHeading text={t('network-policy~Namespace details')} />
```

Comment:
Hi, when checked on network policy detail page, found this bug: https://bugzilla.redhat.com/show_bug.cgi?id=1891419 , so the translation text also need update accordingly.

Reply:
Hi @yanpzhan, PTAL and retest with new 'Network policy details' title (per UXD and I18n aggrement on sentance-case headings).  -thanks

Reply:
Hi, @yanpzhan can you retest? The heading should be fixed now.

Reply:
Sorry for late response, I'm on leave yesterday. I checked on console against the pr,  the network policy detail page has been fixed.